.github/workflows/integration-tests.yml

```diff
@@ -275,8 +275,10 @@ jobs:
           LOKI_BASIC_AUTH: ${{ secrets.LOKI_BASIC_AUTH }}
           LOGSTREAM_LOG_TARGETS: ${{ vars.LOGSTREAM_LOG_TARGETS }}
           GRAFANA_URL: ${{ vars.GRAFANA_URL }}
+          BASE64_CONFIG_OVERRIDE: "override-me"
         run: |
           echo ::add-mask::$CHAINLINK_IMAGE
+          echo ::add-mask::$BASE64_CONFIG_OVERRIDE
```

Comment:
@Tofel btw, what's inside decoded `BASE64_CONFIG_OVERRIDE`? Is it toml with `key=value`? If yes, please note that `value` will not be masked in github logs.

Reply:
It's not key-value pairs. It can be a small part of the config or it can be whole config. In this very case it's what's not defined in general/product defaults and needs be dynamically set (like passwords or selected network).